module-bluetooth/Bluetooth/CommandHandler.cpp

```diff
@@ -39,7 +39,7 @@ namespace bluetooth
         this->driver->registerPowerOnCallback([profilePtr = this->profileManager]() { profilePtr->init(); });
     }
 
-    Error::Code CommandHandler::handle(Command command)
+    Error::Code CommandHandler::handle(Command &command)
```

Comment:
can this switch case be removed with proper variant use? :)

Reply:
I don't think so - some commands do not use additional data, so there's no way to distinguish commands via data type